.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+repos:
+
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.5.0
+    hooks:
+    -   id: check-docstring-first
+    -   id: check-json
+    -   id: check-yaml
+    -   id: debug-statements
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+
+- repo: https://gitlab.com/pycqa/flake8
```

Comment:
The spacing for this section looks different than the other two. Let's make the formatting consistent.